app/grandchallenge/api/urls.py

```diff
@@ -158,34 +155,22 @@
 )
 router.register(r"workstations/sessions", SessionViewSet)
 
-schema_view = get_schema_view(
-    openapi.Info(
-        title=f"{settings.SESSION_COOKIE_DOMAIN.lstrip('.')} API",
-        default_version="v1",
-        description=f"The API for {settings.SESSION_COOKIE_DOMAIN.lstrip('.')}.",
-        license=openapi.License(name="Apache License 2.0"),
-        terms_of_service=reverse_lazy(
-            "policies:detail", kwargs={"slug": "terms-of-service"}
-        ),
-    ),
-    public=True,
-    permission_classes=(permissions.AllowAny,),
-    patterns=[
+
+class SchemaView(SpectacularAPIView):
+    urlconf = [
         path("api/v1/", include(router.urls)),
-        path("api/v1/metrics/", MetricsAPIView.as_view()),
-    ],
-)
+    ]
+
 
 urlpatterns = [
-    re_path(
-        r"^swagger(?P<format>\.json|\.yaml)$",
-        schema_view.without_ui(),
-        name="schema-json",
-    ),
+    path("schema/", SchemaView.as_view(), name="schema"),
     # Do not namespace the router.urls without updating the view names in
     # the serializers
     path("v1/", include(router.urls)),
     path("v1/metrics/", MetricsAPIView.as_view(), name="metrics"),
-    path("auth/", include("rest_framework.urls", namespace="rest_framework")),
```

Comment:
Did you intentionally remove these `auth` endpoints? I don't know if it was used anywhere?

Reply:
Yes, I'm pretty sure they're only used in the HTML renderer which is only sometimes enabled by developers. We have our own login views that you can use instead.